src/index.spec.ts

```diff
@@ -84,15 +80,30 @@ it('should throw if the validation fails', () => {
   );
 });
 
-it('should describe all registered variables', () => {
+it('should throw if a post validation rule fails', () => {
+  const envaridator = new Envaridator();
+  envaridator.registerPostValidation(
```

Comment:
Its best for post-validation rules not to have a name,

Reply:
Fixed. Removed the name, put the description on first parameter position.